authz/api/http/endpoint_test.go

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package http_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/casbin/casbin/v2"
+	"github.com/casbin/casbin/v2/model"
+	"github.com/mainflux/mainflux/authz"
+	httpapi "github.com/mainflux/mainflux/authz/api/http"
+	"github.com/mainflux/mainflux/users/mocks"
+	"github.com/opentracing/opentracing-go/mocktracer"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	secret      = "secret"
+	contentType = "application/json"
+	email       = "user@example.com"
+	token       = "token"
+	wrongValue  = "wrong_value"
```

Comment:
Is this one used?

Reply:
Also, you have previously something called `wrong`, and here you call it `wrongValue` - stay consistent.